### ISSUE TITLE
Wrist not stopping properly

### ISSUE THREAD

**Issue by 0lionelzhang0** [hello-robot/stretch_body#74]
https://github.com/hello-robot/stretch_body/blob/15c93620da897b5f889c90d6a4832a0e4c7a7437/body/stretch_body/dynamixel_hello_XL430.py#L133-L136

Disable torque before setting hw_valid to False?

**Comment by hello-binit** [comment]
Good catch, thanks! Fixed on our [develop branch](https://github.com/hello-robot/stretch_body/blob/develop/body/stretch_body/dynamixel_hello_XL430.py#L133-L136). Will get pushed as a new release on PyPi soon.
